src/Surfnet/ServiceProviderDashboard/Application/Assembler/ServiceStatusAssembler.php

```diff
@@ -0,0 +1,195 @@
+<?php
+/**
+ * Copyright 2018 SURFnet B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Surfnet\ServiceProviderDashboard\Application\Assembler;
+
+
+use Surfnet\ServiceProviderDashboard\Application\Dto\ServiceStatusDto;
+use Surfnet\ServiceProviderDashboard\Application\Service\ServiceStatusService;
+use Surfnet\ServiceProviderDashboard\Domain\Entity\Service;
+use Symfony\Component\Routing\RouterInterface;
```

Comment:
You certainly are not the first to do this, but we should aim to not couple to the infrastructure layer in the application layer. Without going overboard, can you think of a way to remove this coupling?

Reply:
For example you can probably create the routes in the Twig template.

Reply:
Moved the logic to the service